ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Hey team! 
+
+I was using your awesome geocoding engine when I noticed something interesting.
```

Comment:
I never thought I'd say this, but calling ourselves awesome in a template for reporting an issue _might_ be too much. But it might be okay.